gradle/build-gradle-plugins.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2014-2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Spring XD Gradle plugin projects, supporting builds for modules, etc.
+*/
+
+buildscript {
+	repositories {
+		maven { url 'http://repo.spring.io/plugins-release' }
+		jcenter()
+	}
+	dependencies {
+		classpath("org.springframework.boot:spring-boot-gradle-plugin:1.2.1.RELEASE")
+		classpath("org.springframework.build.gradle:propdeps-plugin:0.0.7")
+	}
+}
+project('spring-xd-module-plugin') {
+	apply plugin: 'maven'
+    description = 'Gradle plugin to build a Spring XD Module'
+	dependencies {
+ 		compile gradleApi()
+ 		compile localGroovy()
+ 		runtime "org.springframework.boot:spring-boot-gradle-plugin:1.2.1.RELEASE"
```

Comment:
I'll look at into inheriting the versions. I think the the buildscript deps aren't technically needed, but we commonly use these in the xd build (e.g., to use provided dependencies if needed). The project deps are needed to package these plugins with the spring-xd-module plugin (i.e., they don't need to be declared in the module project.